plugins/frontend/backstage-plugin-aws/src/components/S3BucketCard/S3BucketCard.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import {
+  InfoCard,
+  InfoCardVariants,
+  StructuredMetadataTable,
+  MissingAnnotationEmptyState,
+} from '@backstage/core-components';
+import { Box, CircularProgress, makeStyles } from '@material-ui/core';
+import { useApi } from '@backstage/core-plugin-api';
+import { Entity } from '@backstage/catalog-model';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { awsApiRef } from '../../api/AwsApi';
+import { ACCOUNT_ID_ANNOTATION, S3_BUCKET_ARN_ANNOTATION } from '../../constants';
+import { parse as parseArn } from '@aws-sdk/util-arn-parser';
+import { useAsync } from 'react-use';
+
+const useStyles = makeStyles(theme => ({
+  infoCard: {
+    marginBottom: theme.spacing(3),
+    '& + .MuiAlert-root': {
+      marginTop: theme.spacing(3),
+    },
+    '& .MuiCardContent-root': {
+      padding: theme.spacing(2, 1, 2, 2),
+    },
+    '& td': {
+      whiteSpace: 'normal',
+    },
+  },
+}));
+
+type Props = {
+  /** @deprecated The entity is now grabbed from context instead */
+  entity?: Entity;
+  variant?: InfoCardVariants;
+};
+
+export const S3BucketCard = (props: Props) => {
```

Comment:
Could you please add a basic test for this ?

Reply:
done.